### PR TITLE
Edited SDES callback log messaging

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -2,8 +2,8 @@ resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefac
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.8.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.1.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.9.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.2.0")
```
Comment (LewisMitchell) [inline project/plugins.sbt:6]: This has been sorted in #37

File: app/controllers/SDESCallbackController.scala [hunk 2]
```diff
@@ -49,7 +49,8 @@ class SDESCallbackController @Inject()(auditService: AuditService, cc: Controlle
               Future(BadRequest("Failed to parse to model"))
             },
             sdesCallback => {
-              logger.info(s"[SDESCallbackController][handleCallback] Callback received from SDES with status ${sdesCallback.notification}. SDESCallback received  = $sdesCallback.toString")
+              logger.debug(s"[SDESCallbackController][handleCallback] Callback received from SDES with status ${sdesCallback.notification}. SDESCallback received  = ${sdesCallback.toString}")
```
Comment (LewisMitchell) [inline app/controllers/SDESCallbackController.scala:52]:
```suggestion
              logger.debug(s"[SDESCallbackController][handleCallback] Callback received from SDES with status ${sdesCallback.notification}. SDESCallback received = ${sdesCallback.toString}")
```

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: app/controllers/SDESCallbackController.scala (hunk 2 above)
Comment (AndrewRButcher) [reply app/controllers/SDESCallbackController.scala]: Fixed ✅

File: project/plugins.sbt (hunk 1 above)
Comment (AndrewRButcher) [reply project/plugins.sbt]: Rolled back ✅

**Review by LewisMitchell — DISMISSED** [review]

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**